openshift-ci/deploy_metallb.sh

```diff
@@ -16,6 +16,16 @@ wait_for_pods() {
       exit 1
     fi
   done
+
+  attempts=0
+  until oc -n $FRRK8S_NAMESPACE wait --for=condition=Ready --all pods --timeout 900s; do
```

Comment:
I am in favor of less bash loop retrial logic, would something like that will work?
```
timeout 5m bash -c 'until oc get csv -n $NAMESPACE $CSV_NAME; do sleep 5; done'
oc wait --for jsonpath='{.status.phase}'=Succeeded csv/"$CSV_NAME" -n "$NAMESPACE" --timeout=300s
```